src/js/model/rules/index.js

```diff
@@ -10,7 +10,7 @@ var dl = require('datalib'),
 
 function rules(prototype) {
 
-  prototype.bind = function(property, id, manual) {
+  prototype.bindProp = function(property, id, manual) {
```

Comment:
As a general rule I recommend against naming methods identically to important built-in constructs; this didn't overwrite anything b/c it was not a function, but it did confuse me for a while.